frameworks/elastic/build-kibana.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
```

Comment:
It gets called by the framework developer, just like `build.sh` in that directory. This build script is specific to Kibana because it's not a SDK-based framework. ❄️